pkg/mcp/snapshot/snapshot.go

```diff
@@ -227,3 +239,54 @@ func (c *Cache) Status(group string) *StatusInfo {
 	}
 	return nil
 }
+
+// Types return all types of snapshots that the server layer is serving.
+func (c *Cache) Types() []string {
+	types := make([]string, 0, len(c.snapshots))
+
+	for t := range c.snapshots {
+		types = append(types, t)
+	}
+	sort.Strings(types)
+
+	return types
+}
+
+// Snapshots return the snapshots information
+func (c *Cache) Snapshots(group string) []Info {
```

Comment:
Snapshot has a particular meaning, so using it here is confusing. Can you rename it to GetSnapshotInfo?